stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/ui/utils/GiphyInfoType.kt

```diff
@@ -47,12 +48,14 @@ public enum class GiphyInfoType(public val value: String) {
 /**
  * Default width used for Giphy Images if no width metadata is available.
  */
-internal const val GIPHY_INFO_DEFAULT_WIDTH_DP: Int = 200
+@InternalStreamChatApi
```

Comment:
This file has been moved to `ui-common` using the same package as it previously did to avoid any breaking changes.

This does however break our `ui-common` structure a bit, so if you want me to make a breaking change here please say so.

Reply:
If we can avoid breaking changes, that's good - structure wise, it's not too big of an issue